scripts/templates/pebble-css.hbs

```diff
@@ -1,31 +1,41 @@
 @font-face {
-font-family: "{{fontName}}";
-src: {{{src}}};
+    font-family: "{{fontName}}";
+    src: {{{src}}};
 }
 
 {{baseSelector}} {
-line-height: 1;
+    line-height: 1;
 }
 
 {{baseSelector}}:before {
-font-family: {{fontName}} !important;
-font-style: normal;
-font-weight: normal !important;
-vertical-align: top;
+    font-family: {{fontName}} !important;
+    font-style: normal;
+    font-weight: normal !important;
+    vertical-align: top;
 }
 
 {{#each codepoints}}
     .{{../classPrefix}}{{@key}}:before {
-    content: "\\{{this}}";
+        content: "\\{{this}}";
     }
 {{/each}}
 
-* {
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+body {
     margin: 0;
     padding: 0;
     box-sizing: border-box;
-    font-family: "Anarock", "Roboto", sans-serif;
+    font-family: "Anarock", -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     -webkit-tap-highlight-color: transparent;
 }
+
+input, textarea, select, button {
```

Comment:
Please don't do this. find another workaround.

Reply:
I thought this would be better than `*` as it is currently.

Reply:
done